crates/astria-sequencer-relayer/src/validator.rs

```diff
@@ -0,0 +1,105 @@
+use std::path::Path;
+
+use eyre::{
+    bail,
+    WrapErr as _,
+};
+use subtle_encoding::bech32;
+use tendermint::account;
+use tracing::instrument;
+use zeroize::{
+    Zeroize,
+    ZeroizeOnDrop,
+};
+
+const BECH32_HUMAN_READABLE_PREFIX: &str = "metrovalcons";
+
+/// `Validator` holds the ed25519 keys to sign and verify tendermint
+/// messages. It also contains its address in the tendermint network
+/// and a bech32 encoded address that is used when communicating with metro.
+/// (FIXME: is this true?).
+#[derive(Debug, Zeroize, ZeroizeOnDrop)]
+pub(crate) struct Validator {
+    /// The tendermint validator account address; defined as
+    /// Sha256(verification_key)[..20].
+    #[zeroize(skip)]
+    pub(crate) address: account::Id,
+    /// The bech32-encoded validator account address with "metrovalcons" as
+    /// as the human readable prefix.
+    ///
+    /// FIXME: Add a note on what this is used for.
+    /// NOTE 1: why apply bech32 to the validator *address* instead of its
+    ///         verification (i.e. public) key? tendermint provides `PublicKey::to_bech32`
+    /// NOTE 2: tendermint's `PublicKey::to_bech32` prepends the pubkey with extra bytes
+    ///         to make it amino compatible. Do we need this too?
+    #[zeroize(skip)]
+    pub(crate) bech32_address: String,
```

Comment:
i think we can remove all bech32 addresses in a follow-up as they're cosmos-specific

Reply:
Opened #97